tests/Functionnal/Om/Repository/JsonModel/JsonRepositoryTest.php

```diff
@@ -28,8 +28,9 @@ public function testFindAll()
         $repository = $this->objectManager->getRepository(DummyJson::class);
 
         $collection = $repository->findAll();
-        $this->assertCount(3, $collection);
-        foreach ($collection as $dummy) {
+        $results = iterator_to_array($collection);
+        $this->assertCount(3, $results);
```

Comment:
Maybe con we use

iterator_count

to count the number of elements instead iterator_to_array

Reply:
iterator_count returns an int and assert count expects a ```Countable|Traversable|array```. I don't see what this function can do in the context?